doozer/doozerlib/runtime.py

```diff
@@ -107,6 +107,24 @@ def __init__(self, **kwargs):
         self.disable_gssapi = False
         self._build_data_product_cache: Model = None
 
+        # init cli options
+        self.group = None
```

Comment:
as we are trying to remove duplications, wouldn't it make sense to move some of these fields into artcommonlib.runtime? e.g. group, images, rpms, are going to be used by elliott as well.